examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin, tmax=tmax,
+                             picks=picks, n_jobs=1)
+psds = 10 * np.log10(psds)
+psds_mean = psds.mean(0).mean(0)
+psds_std = psds.std(0).std(0)
```

Comment:
I think you want to do only std of the channels (not across epochs and channels). Look at plot_epochs_psd in mne/viz/epochs.py (L467)